### PR TITLE
CHE-2330 fix behavior for output channel API

### PR DESCRIPTION
**Pull request by olexii4**
Fix behavior for output channel API (show and hide methods for OutputChannel).

Related issue: https://github.com/theia-ide/theia/issues/2330

Signed-off-by: Oleksii Orel <oorel@redhat.com>

### CODE REVIEW

File: packages/plugin-ext/src/main/browser/output-channel-registry-main.ts [hunk 1]
```diff
@@ -14,21 +14,25 @@
...
-import {OUTPUT_WIDGET_KIND} from '@theia/output/lib/browser/output-widget';
+import {OutputWidget} from '@theia/output/lib/browser/output-widget';
+import {OutputContribution} from '@theia/output/lib/browser/output-contribution';
 import {OutputChannel, OutputChannelManager} from '@theia/output/lib/common/output-channel';
 import {OutputChannelRegistryMain} from '../../api/plugin-api';
 
 export class OutputChannelRegistryMainImpl implements OutputChannelRegistryMain {
-    private delegate: OutputChannelManager;
+    private outputChannelManager: OutputChannelManager;
+    private viewContribution: { openView: (arg: { activate: boolean }) => Promise<OutputWidget> };
```
Comment (akosyakov) [inline packages/plugin-ext/src/main/browser/output-channel-registry-main.ts]: `{ openView: (arg: { activate: boolean }) => Promise<OutputWidget> }` =>`OutputContribution `?

Comment (olexii4) [reply packages/plugin-ext/src/main/browser/output-channel-registry-main.ts]: Just to clarify what we can use. I have fixed it.

**Comment by benoitf** [comment]
@olexii4 seems the commits are not signed-Off (DCO check is failing)

**Comment by olexii4** [comment]
@benoitf thanks a lot. Done

**Comment by olexii4** [comment]
@akosyakov  WDYT?

**Pull request merged**